src/test_l1_dsFPD.c

```diff
@@ -1341,1229 +1341,6 @@ void test_l1_dsFPD_negative_dsGetFPColor (void)
     UT_LOG("\n Out  %s\n",__FUNCTION__);
 }
 
-/**
```

Comment:
For the deprecated api's we can mention as deprecated in the test case as well instead of removing it now.

Reply:
done